rtos/Thread.cpp

```diff
@@ -86,6 +86,12 @@ osStatus Thread::start(Callback<void()> task) {
     memset(&_obj_mem, 0, sizeof(_obj_mem));
     _attr.cb_size = sizeof(_obj_mem);
     _attr.cb_mem = &_obj_mem;
+
+    /* Allow new thread to call secure functions */
+#if (__DOMAIN_NS == 1U)
+    _attr.tz_module = 1U;
```

Comment:
@cyliangtw I believe, we should not do this by default for every thread in non-secure mode, rather non-secure application should set this attribute. It would be good to have this as parameter in constructor. 
@sg- What do you suggest?

Reply:
How does tz_module attribute work? Is it something a user can set freely ? In the runtime?

If answer is yes, can be set in the runtime, and we want them to set it, then there should be setter (via ctor or own method).

Reply:
@0xc0170 - It's effective in OS context switch for including trust-zone context store&load. Ideally, to let user to set the attribute freely in create new Thread, though user can't be easy to judge to enable tz_module or not.